qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -95,6 +95,27 @@
                     reboot_method = "system_reset"
                     when_start = "reboot"
                     after_finished = "verify_alive"
+                - data_integrity:
+                    type = live_snapshot_integrity
+                    drive_cache = writethrough
```

Comment:
Can you please make a comment for this about why it should set `drive_cache = writethrough`?